src/utils/fishData.js

```diff
@@ -396,6 +419,53 @@ const fishComponents = {
         AppMode.FishShort
       ]
     },
+    s1: {
+      src: s1_image,
+      anchor: [70, 50],
+      eyeAnchor: [0, -7],
+      mouthAnchor: [2, 15],
+      pectoralFinBackAnchor: [10, 15],
+      pectoralFinFrontAnchor: [38, 15],
+      //pectoralFinBackAnchor: [30, 55],
```

Comment:
Can we explain why it's commented out, or remove permanently?  Otherwise gets confusing to see later on...

Reply:
Added a comment